doc/modules/compose.rst

```diff
@@ -431,11 +458,11 @@ By default, the remaining rating columns are ignored (``remainder='drop'``)::
 
   >>> column_trans.get_feature_names()
   ... # doctest: +NORMALIZE_WHITESPACE +ELLIPSIS
-  ['city_category__x0_London', 'city_category__x0_Paris', 'city_category__x0_Sallisaw',
-  'title_bow__bow', 'title_bow__feast', 'title_bow__grapes', 'title_bow__his',
-  'title_bow__how', 'title_bow__last', 'title_bow__learned', 'title_bow__moveable',
-  'title_bow__of', 'title_bow__the', 'title_bow__trick', 'title_bow__watson',
-  'title_bow__wrath']
+  ['city_category__city_London', 'city_category__city_Paris', 'city_category__city_Sallisaw',
```

Comment:
This is somewhat backwards-incompatible but maybe not that bad?

Reply:
If we rename get_feature_names, we no longer have the problem of backward incompatibility.